openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2537,6 +2548,127 @@ The following is a non-normative example of a Credential Response containing a C
 
+### Interactive Authorization Endpoint Binding {#iae-binding-mso-mdoc}
+
+To bind the Interactive Authorization Endpoint to a Verifiable Presentation using the Credential Format defined in this section, the `SessionTranscript` CBOR structured as defined in Section 9.1.5.1 in [@ISO.18013-5] MUST be used with the following changes:
+
+* `DeviceEngagementBytes` MUST be `null`.
+* `EReaderKeyBytes` MUST be `null`.
+* `Handover` MUST be the `OpenID4VCIIAEHandover` CBOR structure as defined below.
+
+Note: The following section contains a definition in Concise Data Definition Language (CDDL), a language used to define data structures - see [@RFC8610] for more details. `bstr` refers to Byte String, defined as major type 2 in CBOR and `tstr` refers to Text String, defined as major type 3 in CBOR (encoded in utf-8) as defined in section 3.1 of [@RFC8949].
+
+```cddl
+OpenID4VCIIAEHandover = [
+  "OpenID4VCIIAEHandover", ; A fixed identifier for this handover type
+  OpenID4VCIIAEHandoverInfoHash ; A cryptographic hash of OpenID4VCIIAEHandoverInfo
+]
+
+; Contains the sha-256 hash of OpenID4VCIIAEHandoverInfoBytes
+OpenID4VCIIAEHandoverInfoHash = bstr
+
+; Contains the bytes of OpenID4VCIIAEHandoverInfo encoded as CBOR
+OpenID4VCIIAEHandoverBytes = bstr .cbor OpenID4VCIIAEHandoverInfo
+
+OpenID4VCIIAEHandoverInfo = [
+  iaeUrl,
+  nonce,
+  jwkThumbprint
+] ; Array containing handover parameters
+
+iaeUrl = tstr
+
+nonce = tstr
+
+jwkThumbprint = bstr
+```
+
+The `OpenID4VCIIAEHandover` structure has the following elements:
+
+* The first element MUST be the string `OpenID4VCIIAEHandover`. This serves as a unique identifier for the handover structure to prevent misinterpretation or confusion.
+* The second element MUST be a Byte String which contains the sha-256 hash of the bytes of `OpenID4VCIIAEHandoverInfo` when encoded as CBOR.
+* The `OpenID4VCIIAEHandoverInfo` has the following elements:
+  * The first element MUST be the string representing the Interactive Authorization Endpoint URL of the request as described in (#interactive-authorization-request). It MUST NOT be prefixed with `iar:`.
+  * The second element MUST be the value of the `nonce` request parameter.
+  * For the Response Mode `iar-post.jwt`, the third element MUST be the JWK SHA-256 Thumbprint as defined in [@!RFC7638], encoded as a Byte String, of the Verifier's public key used to encrypt the response. If the Response Mode is `iar-post`, the third element MUST be `null`. For unsigned requests, including the JWK Thumbprint in the `SessionTranscript` allows the Verifier to detect whether the response was re-encrypted by a third party, potentially leading to the leakage of sensitive information. While this does not prevent such an attack, it makes it detectable and helps preserve the confidentiality of the response.  
+
+The following is a non-normative example of the input JWK for calculating the JWK Thumbprint in the context of `OpenID4VCIIAEHandoverInfo`:
```

Comment:
```suggestion
The following is a non-normative example of the input JWK for calculating the JWK Thumbprint in the context of `OpenID4VCIIAEHandoverInfo`:

```

Reply:
It compiled for me locally without the extra lines but I can certainly commit your suggestion.